<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>StudyQuestions</class>
    <widget class="QWidget" name="StudyQuestions">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>StudyQuestions</string>
        </property>
        <widget class="QLabel" name="questionLabel">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>70</y>
                    <width>311</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QLineEdit" name="answerLineEdit">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>140</y>
                    <width>321</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="inputMask">
                <string/>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="placeholderText">
                <string>Answer...</string>
            </property>
        </widget>
        <widget class="QPushButton" name="continueButton">
            <property name="geometry">
                <rect>
                    <x>120</x>
                    <y>210</y>
                    <width>161</width>
                    <height>51</height>
                </rect>
            </property>
            <property name="text">
                <string>Check</string>
            </property>
        </widget>
        <widget class="QLabel" name="correctnessLabel">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>170</y>
                    <width>321</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>22</y>
                    <width>331</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="value">
                <number>24</number>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
